test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1216,6 +1221,207 @@
+		<command>$JAVA_EXE$ $currentMode$ -XX:-ShareAnonymousClasses -XX:-ShareUnsafeClasses $EXPORTS$ $CP_HANOI$ $PROGRAM_UNSAFE$</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">test done!</output>
+
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 65-b: Make sure none of the classes has been stored in the cache" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printstats=orphan</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">ORPHAN: org/openj9/test/classtests/ClassTest at</output>
+	    
+		<output type="failure" caseSensitive="yes" regex="yes" javaUtilPattern="yes">ORPHAN: org/openj9/test/classtests/AnonClassAndUnsafeClassTest/(00000000|0000000000000000) at</output>
+		<output type="failure" caseSensitive="yes" regex="yes" javaUtilPattern="yes">ORPHAN: org/openj9/test/classtests/AnonClassAndUnsafeClassTest at</output>		
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+	</test>
+
+	<test id="Test 65 clean-up" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,destroy</command>
+		<output type="success" caseSensitive="yes" regex="no">Cache does not exist</output>
+		<output type="success" caseSensitive="yes" regex="no">has been destroyed</output>
+		<output type="success" caseSensitive="yes" regex="no">is destroyed</output>
+		
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 66-a: Store the Anonymous Class and the Unsafe Class in the cache for tests 66-b and c" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $EXPORTS$ $currentMode$ $CP_HANOI$ $PROGRAM_UNSAFE$</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">test done!</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">Test</output>
+		<output type="required" caseSensitive="yes" regex="yes" javaUtilPattern="yes">Test</output>
```

Comment:
This second output of "Test" being the same as the first doesn't work. Better to have the anon and unsafe class output a different string.